docs/advanced-topics/docker-alternatives.rst

```diff
@@ -0,0 +1,130 @@
+Docker Alternatives
+===================
+
```

Comment:
This could use a note like that at the beginning of https://docs.dockstore.org/en/develop/getting-started/dockstore-workflows.html to indicate that this tutorial is more experimental than many others and we're working on integrations to make things easier.